Dart/src/com/jetbrains/lang/dart/analyzer/DartAnalysisServerService.java

```diff
@@ -104,6 +113,11 @@
   private final AnalysisServerListener myAnalysisServerListener = new AnalysisServerListenerAdapter() {
 
     @Override
+    public void computedAnalyzedFiles(List<String> directories) {
```

Comment:
Looks like a misleading parameter name in server API. The list contains file paths, not directories.